<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>725</width>
    <height>475</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>512</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Depth to Water for Water4</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <pointsize>18</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Preliminary Results:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextBrowser" name="resultsTextBox">
        <property name="toolTip">
         <string>Display of preliminary results from the VES analysis</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1">
     <layout class="QVBoxLayout" name="mplvl">
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="2">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>Use a negative sign (-) for coordinates that are South or West</string>
        </property>
        <property name="text">
         <string>Coordinates (use negative values for South and West)</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Longitude:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="longitudeLineEdit">
        <property name="toolTip">
         <string>Enter longitude (- sign for West)</string>
        </property>
        <property name="inputMask">
         <string>#000.00000</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Latitude:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="latitudeLineEdit">
        <property name="toolTip">
         <string>Enter latitude (- sign for South)</string>
        </property>
        <property name="inputMask">
         <string>#00.00000</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QDateTimeEdit" name="dateTimeEdit">
        <property name="toolTip">
         <string>Enter current time and date if incorrect</string>
        </property>
        <property name="dateTime">
         <datetime>
          <hour>0</hour>
          <minute>0</minute>
          <second>0</second>
          <year>2000</year>
          <month>1</month>
          <day>1</day>
         </datetime>
        </property>
        <property name="displayFormat">
         <string>yyyy-M-d hh:mm </string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
        <property name="currentSectionIndex">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>rhoB</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="2">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>rhoM</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>rhoA</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="rhoALineEdit">
        <property name="toolTip">
         <string>Adjust rhoA value</string>
        </property>
        <property name="inputMask">
         <string>000.00</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLineEdit" name="rhoMLineEdit">
        <property name="toolTip">
         <string>Adjust rhoM value</string>
        </property>
        <property name="inputMask">
         <string>000.00</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="rhoBLineEdit">
        <property name="toolTip">
         <string>Adjust rhoB value</string>
        </property>
        <property name="inputMask">
         <string>000.00</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" colspan="2">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>dA</string>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QLineEdit" name="dALineEdit">
        <property name="toolTip">
         <string>Adjust dA value</string>
        </property>
        <property name="inputMask">
         <string>000.00</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" colspan="5">
       <widget class="QPushButton" name="pushButton">
        <property name="font">
         <font>
          <pointsize>14</pointsize>
         </font>
        </property>
        <property name="toolTip">
         <string>If satisfied with the preliminary results, click for an estimate of drilling feasibility</string>
        </property>
        <property name="text">
         <string>Drill or no drill?</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="5">
       <widget class="QPushButton" name="pushButton_2">
        <property name="toolTip">
         <string>Rerun the analysis after fine tuning some parameters</string>
        </property>
        <property name="text">
         <string>Rerun VES Inverse Analysis with New Parameters</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
   <zorder>figureViewWidget</zorder>
   <zorder>pushButton_3</zorder>
   <zorder>figureViewWidget</zorder>
   <zorder>figureViewWidget</zorder>
   <zorder>widget</zorder>
   <zorder>lineEdit</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>725</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="separator"/>
    <addaction name="actionSave_as_CSV"/>
    <addaction name="separator"/>
    <addaction name="actionSave_table_as_csv"/>
    <addaction name="actionSave_plot_as_png"/>
    <addaction name="separator"/>
    <addaction name="actionExit_2"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionOne_Window"/>
    <addaction name="actionMulti_Windows"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionWebsite"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionFiles">
   <property name="text">
    <string>Files</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionOne_Window">
   <property name="text">
    <string>One Window</string>
   </property>
  </action>
  <action name="actionMulti_Windows">
   <property name="text">
    <string>Multi Windows</string>
   </property>
  </action>
  <action name="actionWebsite">
   <property name="text">
    <string>Website</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSave_as_CSV">
   <property name="text">
    <string>Save project</string>
   </property>
  </action>
  <action name="actionSave_as_docx">
   <property name="text">
    <string>Save table as .csv</string>
   </property>
  </action>
  <action name="actionSave_table_as_csv">
   <property name="text">
    <string>Save table as .csv</string>
   </property>
  </action>
  <action name="actionSave_table_as_xlsx">
   <property name="text">
    <string>Save table as .xlsx</string>
   </property>
  </action>
  <action name="actionSave_plot_as_png">
   <property name="text">
    <string>Save plot as .png</string>
   </property>
  </action>
  <action name="actionExit_2">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
